.github/CODEOWNERS

```diff
@@ -5,30 +5,42 @@
 
 # Tendermint2 (Gno version).
 tm2/*                        @gnolang/tech-staff
-tm2/pkg                      @jaekwon @moul
+tm2/pkg                      @jaekwon @piux2 @moul
 # TODO: add per package exceptions
 
-
 # Docs & Content.
 docs/                        @gnolang/devrels @gnolang/tech-staff
 misc/docusaurus/             @gnolang/devrels @gnolang/tech-staff
 # TODO: add non-tech people here.
 
-
 # Gno examples and default contracts.
-examples/*                   @gnolang/tech-staff
+examples/*                   @gnolang/tech-staff @gnolang/devrels
 # TODO: add people from the community here.
 
-
 # Gno.land.
-gno.land/*                   @gnolang/tech-staff
-
+gno.land/*                   @moul @gnolang/tech-staff
```

Comment:
```suggestion
gno.land/*                   @moul
gno.land/pkg/integration     @gfanton
```

Reply:
I'm thinking about this suggestion, and not sure what the right approach is here. On one hand we want the team to own the chain part init of the codebase, on the other we don't have so many mission critical tools here that warrant entire team ownership (especially the initial genesis config)

Reply:
I was considering the proposal when creating the PR. We can try it this way, and I'll ask for help if I become overwhelmed.